Adamant/Modules/CoinsNodesList/ViewModel/CoinsNodesListState.swift

```diff
@@ -17,7 +17,7 @@ struct CoinsNodesListState: Equatable {
     static var `default`: Self {
         Self(
             sections: .init(),
-            fastestNodeMode: false,
+            fastestNodeMode: true,
```

Comment:
But it's just a default value of the state to be represented 🧐
I am not sure if the problem is actually solved

Reply:
seems to me that changed should be done here:

```
public extension NodeGroup {
    var defaultFastestNodeMode: Bool {
        switch self {
        case .adm:
            return false
        case .eth, .doge, .dash, .btc, .klyNode, .klyService, .ipfs, .infoService:
            return true
        }
    }
}
```

Reply:
It's the default value of a state, the fix is definitely has to be in some other place